tests/FixedArrayTest.php

```diff
@@ -166,6 +166,65 @@ public function providerShiftDown() : array
         ];
     }
 
+    public function testCreate()
+    {
+        $fixedArray = FixedArray::fromArray([1, 2, 3, 4]);
+        FixedArray::create();
+
+        $this->assertSame(4, $fixedArray->getSize());
+        $this->assertInstanceOf(FixedArray::class, $fixedArray);
+    }
+
+    public function testToArray()
+    {
+        $expectedArray = [1, 2, 3, 4];
+        $fixedArray = FixedArray::fromArray($expectedArray);
+        $result = $fixedArray->toArray();
+
+        $this->assertCount(4, $result);
+        $this->assertSame($expectedArray[0], 1);
+        $this->assertSame($expectedArray[1], 2);
+        $this->assertSame($expectedArray[2], 3);
+        $this->assertSame($expectedArray[3], 4);
```

Comment:
You can replace these 6 lines with `assertSame($expectedArray, $fixedArray->toArray())`